### ISSUE TITLE
Request for CNA 

### ISSUE THREAD

**Issue by ProDigySML** [distributedweaknessfiling/DWF-CNA-Registry#2]
Hi, 
I am the head of Bulletproof ZDS (Zero Day Squad), a new team of Zero Day hunters. We hunt for CVEs in open source projects, alongside commercial projects and also get some requests for helping with CVE assignments through connections. We are an Australian company and are currently trying to grow in size.
Please do let me know if any more information is required.
Sajeeb

**Comment by telnet404** [comment]
:)  any website?

**Comment by ProDigySML** [comment]
No. We are working on getting that fixed up. But we have a few CVEs to our name and are trying to get a fair few more approved.

**Comment by ProDigySML** [comment]
Our website is https://0dd.zone. We have released more than 50 CVEs this year and intend to apply for another 24 very soon. I was wondering if there were any updates on the progress of this application @kurtseifried
